src/cloud/docker/docker-containers-service.md

```diff
@@ -11,21 +11,53 @@ functional_areas:
 The following containers provide the services required to build, deploy and run {{site.data.var.ee}} sites.
 
 {:.bs-callout-info}
-See the [service version values available]({{ site.baseurl }}/cloud/docker/docker-containers.html#service-versions) for use when launching Docker.
+See the [service version values available]({{site.baseurl}}/cloud/docker/docker-containers.html#service-versions) for use when launching Docker.
 
-## Database container
+## Database Container
```

Comment:
I think it is completely fine to define the container here, but adding the various database procedures makes the topic too long.